R/buoyancy.freq.R

```diff
@@ -2,6 +2,40 @@
 # Buoyancy Frequency
 # Author: Luke Winslow <lawinslow@gmail.com>
 #
+
+
+#' Calculates buoyancy frequency.
+#' 
+#' Calculate the buoyancy frequency (Brunt-Vaisala frequency) for a temperature
+#' profile.
+#' 
+#' 
+#' @param wtr a numeric vector of water temperature in degrees C
+#' @param depths a numeric vector corresponding to the depths (in m) of the wtr
+#' measurements
+#' @return Returns a vector of buoyancy frequency in units \code{sec^-2}.
+#' Return value has attribute "depths" which define buoyancy frequency depths
+#' (which differ from supplied depths).
+#' @author Luke Winslow

```

Comment:
While you're rooting around the documentation, I personally think these @author tags for the functions specifically should be dropped. We aren't consistent with usage and most of them are out of date. @jread-usgs do you agree?

Reply:
FWIW, I agree too as it adds another thing to maintain.

Reply:
These are now dropped throughout.